vehicle/OVMS.V3/components/vehicle_nissanleaf/src/vehicle_nissanleaf.cpp

```diff
@@ -2037,7 +2042,8 @@ OvmsVehicle::vehicle_command_t OvmsVehicleNissanLeaf::CommandWakeup()
   if (!m_enable_write) return Fail; // Disable commands unless canwrite is true
   ESP_LOGI(TAG, "Sending Wakeup Frame");
   unsigned char data = 0;
-  m_can1->WriteStandard(0x679, 1, &data);
+  m_can1->WriteStandard(0x679, 1, &data); //Tops up the 12V battery if connected to EVSE
+  m_can1->WriteStandard(0x5C0, 8, &data); //Wakes up the VCM (by spoofing empty battery request heating)
```

Comment:
Noticed while debugging issues preventing climate control, that this seems to transmit garbage data. 8 bytes sent from the single unsigned char. Could this cause the intermittent problems?

Reply:
(referring to the issue #804 )